src/wrover_status_light/ServiceIntro.md

```diff
@@ -28,20 +28,27 @@ Here is a list of key components involved in services:
 
 While both services and topics work to allow communication between nodes, they differ in terms of their data exchange patterns and the types of interactions they support. 
 
-Unlike topics, which operate asynchronously, service calls provide synchronous communication. The client node waits for the server node to process the request and receive a response before continuing its execution. This synchronous behavior is beneficial in cases where a response is required before proceeding with the next steps of a program.  It is not acceptable for a server to not exist or to never return a response; these will generate exceptions for the service client to handle.
+Unlike topics, which operate asynchronously, service calls provide synchronous communication. The client node waits for the server node to process the request and receives a response before continuing its execution. This synchronous behavior is beneficial in cases where a response is required before proceeding with the next steps of a program.  It is not acceptable for a server to not exist or to never return a response; these will generate exceptions for the service client to handle.
 
 In addition, topics are designed for data streaming and broadcasting continuous streams of information, such as sensor data or robot state updates. They are ideal when nodes require a constant flow of information and do not require immediate responses. On the other hand, services are typically used for accessing functionality that is provided by another node. They are well-suited for situations where nodes need to invoke specific actions or request information from another node, and a response is expected.
 
 ## Defining a Service
-Services are defined by srv files, which are identical to msg files in terms of formatting, and should be located in the `/srv` folder in a package. In this example, the service takes in a `string` and an `int` as input and returns a `float` as output. You can find an example service file at `srv/DoubleService.srv`.
+Services are defined by srv files, which are similar to msg files in terms of formatting except they contain two parts:
 ```
 #request fields
-string input1
-int32 input2
+insert your fields for request messages here as follows:
+[data type] [field name]
+[data type] [field name]
+...
 ---
 #response fields
-float32 output
+insert your fields for response messages here as follows:
+[data type] [field name]
+[data type] [field name]
+...
 ```
+Fields are listed in the srv file each on its individual line, with a `---` separating the request fields from the response fields. Both requests and responses can have a single field, multiple fields, or no fields at all. All srv files for a module should be located in the `/srv` folder for proper complication. 
```

Comment:
Could you also explain here how constants could be defined? I see that in ServiceLight.srv, DONE and NOT_DONE are defined. I'm assuming these are used as constants which can be assigned to the request field?

Reply:
Added in [a470552](https://github.com/WisconsinRobotics/WRoverPlayground/pull/9/commits/a47055254bc5a7390410e548bba23e97ab80c470).